eth_test_parser/src/trie_builder.rs

```diff
@@ -108,7 +107,6 @@ impl TestBody {
     fn get_storage_tries(&self) -> Vec<(H160, PartialTrie)> {
         self.pre
             .iter()
-            .filter(|(_, pre_acc)| !pre_acc.code.0.is_empty())
```

Comment:
I'm not sure if it's strictly safe to assume that an account without code won't have storage either? EIP-158 mentions that as a "pathological" case, though I don't know how that would happen (it might require an address collision?). Besides, it seems valid for the zkEVM to access the (empty) storage trie of an EOA, even if it doesn't have to (it could optionally check just the hash). I think it's best to just pass along all the data with no assumptions about what the zkEVM will need.

Reply:
Yeah I agree. We're only saving on negligible performance here by keeping the filter in, and removing this will ensure that it's included if it's there.